lua/core/norns.lua

```diff
@@ -237,3 +237,8 @@ end
 _startup = function()
   require('core/startup')
 end
+
+_post_startup = function()
+   print('_norns._post_startup')
+   hook.system_post_startup()
```

Comment:
i get slightly weirded out by references to a module in a file which doesn’t import it. i realize that pattern is all over the core code. iirc the `mods` does import `hooks` but i double it is importing it globally and calling it `hook`.

Reply:
yea i agree. but we constantly assume the global names assigned in `startup`. hppy to do my part to change that style

Reply:
ACTUally nvm i am locally importing `hook` b/c i am a good boy

Reply:
actually nvm hook is a local alias, i'm a good boy

Reply:
sure enough. sorry for the false alarm (hard to review on the phone)